package.json

```diff
@@ -5,6 +5,7 @@
   "repository": "github/task-lists-element",
   "main": "dist/index.umd.js",
   "module": "dist/index.esm.js",
+  "types": "task-lists-element.d.ts",
```

Comment:
Should we standardize on `index.d.ts` to match TypeScript's defaults?

Reply:
Sounds good ✨